packages/one-bx-shell-app/src/assets/config/plugins-bak.json

```diff
@@ -0,0 +1,164 @@
+[
+  {
+    "name": "contentReq",
+    "provider": "Ariba",
+    "version": "1.0.1",
+    "uri": "http://localhost:4201/remoteEntry.js",
+    "modules": [
+      {
+        "name": "prList",
+        "type": "angular-ivy-component",
+        "exposedModule": "./PrList",
+        "componentName": "PrListComponent"
+      },
+    ],
+    "components": {
+      "prlist": {
+        "name": "PrListComponent",
+        "module": "./PrList",
+        "entry": "/remoteEntry.js",
```

Comment:
When it is iFrame it will not have a entry, it will just have this index.html or some entry URL. The same should apply for other attributes like module.

Reply:
This covered in iframe module typings, yes we discussed it

Reply:
I am not sure if I follow. when I define iFrame endpoint some url for the app e.g. https://amazon.com/shoppingcard.  what typings it is ? 

There is no Module federation for iFrames, this should be just an entry that points to URL that shoudl be loaded right ?